posthog/tasks/exports/image_exporter.py

```diff
@@ -188,10 +188,10 @@ def export_image(exported_asset: ExportedAsset) -> None:
                 # NOTE: Dashboards are regularly updated but insights are not
                 # so, we need to trigger a manual update to ensure the results are good
                 with conversion_to_query_based(exported_asset.insight):
-                    process_query_model(
+                    process_query_dict(
                         exported_asset.team,
                         exported_asset.insight.query,
-                        limit_context=LimitContext.EXPORT,
+                        limit_context=LimitContext.QUERY_ASYNC,
```

Comment:
This also is important to adjust, otherwise no cache will be populated for the subsequent insight screenshot.